config/services.yaml

```diff
@@ -10,6 +10,8 @@ services:
             $programmeMaxTimeInMinutes: '%env(PROGRAMME_MAX_TIME_IN_MINUTES)%'
             $maxPerPage: '%env(MAX_PER_PAGE)%'
             $apiVersion: '%env(API_VERSION)%'
+            $handlerToImportfrom: '%env(HANDLER_IMPORT_FROM)%'
```

Comment:
Nitpicking here, but if you want to use camel case properly, `From` should also start with an uppercase letter :stuck_out_tongue:

Reply:
Woops